lib/rules/no-jquery-methods.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @fileOverview Disallow the use of specific jQuery methods.
+ * @author Marc Lynch
+ */
+'use strict';
+
+/**
+ * Determines if this expression matches the method blacklist invocation.
+ * @param {ASTNode} node The identifier node.
+ * @returns {Boolean} Returns true if the expression matches, otherwise false.
+ */
+
+const BLACKLIST = [
```

Comment:
This list would be easier to read if each were on a separate line. Can we do this?